File: domain/src/main/java/no/nav/melosys/domain/person/familie/IkkeOmfattetFamilie.java
package no.nav.melosys.domain.person.familie;

public class IkkeOmfattetFamilie {
    private final String uuid;
    private final String begrunnelse;
    private final String begrunnelseFritekst;

    public IkkeOmfattetFamilie(String uuid, String begrunnelse, String begrunnelseFritekst) {
        this.uuid = uuid;
        this.begrunnelse = begrunnelse;
        this.begrunnelseFritekst = begrunnelseFritekst;
    }

    public String getUuid() {
        return uuid;
    }

    public String getBegrunnelse() {
        return begrunnelse;
    }

    public String getBegrunnelseFritekst() {
        return begrunnelseFritekst;
    }
}

File: domain/src/main/kotlin/no/nav/melosys/domain/person/familie/IkkeOmfattetFamilie.kt
package no.nav.melosys.domain.person.familie

data class IkkeOmfattetFamilie(
    val uuid: String,
    val begrunnelse: String?,
    val begrunnelseFritekst: String?
) {
    var sammensattNavn: String? = null
    var ident: String? = null
}
